lib/puppet/provider/cisco_interface_ospf/cisco.rb

```diff
@@ -31,29 +31,28 @@
 
   mk_resource_methods
 
-  # Getter properties that need to generate the getter functions
-  GETTER_PROPS = [
-    :hello_interval, :dead_interval
+  INTF_OSPF_NON_BOOL_PROPS = [
+    :cost,
+    :hello_interval,
+    :dead_interval,
+    :message_digest_key_id,
+    :message_digest_algorithm_type,
+    :message_digest_encryption_type,
+    :message_digest_password,
+    :area,
```

Comment:
I am not sure why it was done this way. I retried after reordering, and it seems ok. On the node, the CLI also seems ok without any order issues. May be 'area' property was added later and it was appended in the end? Anyway, took care of it.